generator/src/googleapis/codegen/template_objects.py

```diff
@@ -233,6 +284,22 @@ def fullClassName(self):  # pylint: disable=g-bad-name
     return MarkSafe(self.RelativeClassName(None))
 
   @property
+  def lowerClassName(self):  # pylint: disable=g-bad-name
```

Comment:
Do we still need this function? I think it was converting from underscores to camel cases?

Reply:
The underscore removal is part of the logic and it is not necessary anymore - I removed it. We will still need it because the template was [previously relying](https://github.com/googleapis/google-api-java-client-services/blob/9a2d3b0f586d6c63a6d1d28b188f06ab5d689261/generator/src/googleapis/codegen/languages/java/2.0.0/templates/_resource.tmpl#L12) on `codeName`, which does work if the class is _not_ duplicated. However, when we compute a de-duplicated class name, `codeName` won't change and we would get a mismatch such as the one pointed here:

> Looking at the [generated changes](https://github.com/googleapis/google-api-java-client-services/pull/16576/files#diff-3d8702c3cc2689ae262c02b827f139369f38e3538658d77b07263ade345d4e58), something like this is not ideal
> 
> ```
> public Accounts1 accounts() {
>       return new Accounts1();
>     }
> ```
> 
> as `Accounts1` is on the surface, initially I thought it's only used internally. If it's on the surface and customer could actually use it, then we may want to be more careful about the naming. I see two problems currently:
> 
> 1. `Accounts1` also does not mean much to customers and they may be confused why there is a `1` appended to it.
> 2. The method name and the class name do not match. `accounts()` actually returns `Accounts1`.
>    Can you spend a little more time and see if there is a way to give more meaningful names?

Reply:
Got it thanks!